src/display.rs

```diff
@@ -99,6 +114,50 @@ pub fn expand(input: &DeriveInput, trait_name: &str) -> Result<TokenStream> {
     })
 }
 
+fn trait_name_to_attribute_name(trait_name: &str) -> &'static str {
```

Comment:
Consider to use smart well-known abbreviations, like `attr` instead `attribute`.